finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -128,11 +128,19 @@ object HighResTimer {
    *
    * It is a shared resource and as such, `stop` is ignored.
    */
-  val Default: com.twitter.util.Timer =
-    new JavaTimer(true, Some("HighResTimer")) {
-      override def stop(): Unit = ()
-    }
+  val Default = new JavaTimer(true, Some("HighResTimer")) {
+    override def stop(): Unit = ()
+    def stopTimer(): Unit = super.stop()
+  }
+
 
+  /**
+   * Stop default HighResTimer
+   */
+  def stop() {
+    Default.stopTimer()
```

Comment:
can we also add a log message warning users that this might break Finagle?  we did something similar when we pulled in your other patch:

https://github.com/twitter/finagle/commit/fa4899e1ba1e9b612dbf7428b0f457ae3855b5b8

Reply:
Done